ocaml/idl/datamodel_certificate.ml

```diff
@@ -64,8 +64,16 @@ let t =
       ; field ~qualifier:StaticRO ~lifecycle ~ty:DateTime "not_after"
           ~default_value:(Some (VDateTime Date.never))
           "Date before which the certificate is valid"
-      ; field ~qualifier:StaticRO ~lifecycle ~ty:String "fingerprint"
+      ; field ~qualifier:StaticRO
+          ~lifecycle:
+            [(Published, rel_stockholm, ""); (Deprecated, "24.18.0", "")]
```

Comment:
We got a new xapi release, so this needs to be updated, otherwise the PR is good to go:

```suggestion
            [(Published, rel_stockholm, ""); (Deprecated, "24.19.0", "")]
```